.github/workflows/acceptance-tests-runner.yml

```diff
@@ -103,6 +103,10 @@ on:
         required: true
       mongodb_atlas_private_key:
         required: true
+      mongodb_atlas_public_key2:
```

Comment:
what about a better naming to indicate its purpose?

Reply:
Not sure. If we use it for other things the generic name might be an advantage.

On the other hand, a more specific name could look like:
`mongodb_atlas_public_key_least_privilege`
or
`mongodb_atlas_public_key_read_only`.
WDYT?

Reply:
`mongodb_atlas_public_key_read_only` is the one I'd vote

Reply:
renamed in 4a5352ee3040f80a247f288799a5637dc1fc26c0